ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -910,10 +912,14 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     
     NSString *progressLabel = nil;
     if ([self shouldDisplayProgressLabel]) {
-        ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
-        
-        if (progress.total > 0) {
-            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,ORKLocalizedStringFromNumber(@(progress.current + 1)), ORKLocalizedStringFromNumber(@(progress.total))];
+        if (viewController.parentReviewStep && viewController.parentReviewStep.isStandalone) {
+            progressLabel = viewController.parentReviewStep.title;
```

Comment:
If it is safe to move the progress label building, we won't need the `hideProgressLabel` property anymore. Instead `shouldDisplayProgressLabel` checks the following `!(self.currentStepViewController.parentReviewStep.isStandalone)`

I think embedded review steps won`t need a special title, since they have a valid progress which should give the user enough context.